src/app/views/transcription.py

```diff
@@ -50,28 +55,29 @@ def importAudioClicked(self):
 
 
     @pyqtSlot()
-    def transcribeClicked(self):
+    def onTranscribeClicked(self):
         self.transcriptionResult.setText('Transcribing...')
         self.setDisabled(True)
-        # The UI update is scheduled after the ending of the slot, that's why we need to force the porcessing of events.
-        QApplication.processEvents()
 
-        try:
-            config = {
-                'encoding' : self.encoding.currentText(),
-                'sampleRate' : self.sampleRate.value(),
-                'languageCode' : self.language.currentText(),
-                'model' : self.model.currentText(),
-                'enhanced' : self.enhanced.checkState()}
-
-            transcriptionResult = SpeechToText.resquestTranscription(serviceAccountPath=self.window().getSetting('serviceAccountPath'),
-                                                                        audioDataPath=self.audioDataPath.text(),
-                                                                        config=config)
-            self.transcriptionResult.setText(transcriptionResult)
-        
-        except Exception as e:
-            self.window().emitToExceptionsManager(e)
-            self.transcriptionResult.setText('')
-        finally:
-            self.setDisabled(False)
-        
\ No newline at end of file
+        config = {'audioDataPath' : self.audioDataPath.text(),
+                    'encoding' : self.encoding.currentText(),
+                    'enhanced' : self.enhanced.checkState(),
+                    'languageCode' : self.language.currentText(),
+                    'model' : self.model.currentText(),
+                    'outputFolder' : self.window().getSetting('defaultOutputFolder'),
+                    'sampleRate' : self.sampleRate.value(),
+                    'serviceAccountPath' : self.window().getSetting('serviceAccountPath')}
```

Comment:
Would be better like this:

    config = {
        'audioDataPath' : self.audioDataPath.text(),
        'encoding' : self.encoding.currentText(),
        'enhanced' : self.enhanced.checkState(),
        'languageCode' : self.language.currentText(),
        'model' : self.model.currentText(),
        'outputFolder' : self.window().getSetting('defaultOutputFolder'),
        'sampleRate' : self.sampleRate.value(),
        'serviceAccountPath' : self.window().getSetting('serviceAccountPath')
    }